core/model/modx/processors/system/dashboard/widget/feed.class.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * Class modDashboardWidgetFeedProcessor
+ *
+ * Used to load the news and security feeds on the dashboard over AJAX. The processed feed content
+ * (i.e. HTML) is returned in object->html.
+ */
+class modDashboardWidgetFeedProcessor extends modProcessor
+{
+    /**
+     * @var modRSSParser
+     */
+    protected $rss;
+
+    public function process()
+    {
+        $feed = $this->getProperty('feed', 'news');
+        if (!in_array($feed, array('news', 'security'), true)) {
+            return $this->failure('Invalid feed type');
+        }
+
+        $enabled = $this->modx->getOption('feed_modx_' . $feed . '_enabled', null, true);
+        if (!$enabled) {
+            return $this->failure();
```

Comment:
No, i'm fine with it (we can never assume only core components would use it).
I was just concerned about the "failure" semantic...